--- build/config/compiler/BUILD.gn.orig	2025-12-08 22:11:59.327391066 -0500
+++ build/config/compiler/BUILD.gn	2025-12-08 22:13:18.691902917 -0500
@@ -694,7 +694,7 @@
 
     # TODO(crbug.com/40283598): This causes binary size growth and potentially
     # other problems.
-    if (default_toolchain != "//build/toolchain/cros:target") {
+    if (false && default_toolchain != "//build/toolchain/cros:target") {
       cflags += [
         "-mllvm",
         "-split-threshold-for-reg-with-hint=0",
